spec/javascript/components/applications/__snapshots__/ApplicationPage.test.js.snap

```diff
@@ -557,7 +557,7 @@ Array [
                     Has ADA Priorities Selected
                   </h4>
                   <p>
-                    Vision impaired;Mobility impaired;Hearing impaired
+                    Vision impairments;Mobility impairments;Hearing impairments
```

Comment:
@akegan This is fine in that it is the direct value from Salesforce and it matches the formatting that already existed here, but perhaps we can add spaces after the colons to make it look a little nicer and be easier to read?

Reply:
this one https://www.pivotaltracker.com/n/projects/1405352/stories/162028377.

Reply:
ok, well looks like both of those cover different aspects of app page formatting, so seems like we've already planned to do this at a later time 👍

Reply:
oh whoops. didn't see ADA priorities in the other one.